from_cpython/Include/object.h

```diff
@@ -826,7 +826,7 @@ PyAPI_FUNC(void) _Py_AddToAllObjects(PyObject *, int force) PYSTON_NOEXCEPT;
 
 
 #define Py_INCREF(op) ((void)(op))
-#define Py_DECREF(op) ((void)(op))
+#define Py_DECREF(op) asm volatile("" : : "x"(op))
```

Comment:
hah, thanks for catching that.  The page detailing constraints that I'd looked at indeed had a capital X.  it just looked exactly like a lowercase x.